MediaBrowser.Model/Configuration/LibraryOptions.cs

```diff
@@ -25,6 +25,8 @@ public LibraryOptions()
             SeasonZeroDisplayName = "Specials";
         }
 
+        public bool Disabled { get; set; }
```

Comment:
You can just assign a default value.

```suggestion
        public bool Enabled { get; set; } = true;
```